src/regex.cr

```diff
@@ -275,14 +305,13 @@ class Regex
     String.build do |result|
       str.each_byte do |byte|
         case byte.unsafe_chr
+        # See `Regex::SPECIAL_CHARACTERS`
         when ' ', '.', '\\', '+', '*', '?', '[',
```

Comment:
You can use `{{*SPECIAL_CHARACTERS}}` here to avoid copying :)

Reply:
Nifty, thx!